### PR TITLE
PORTALS-660

### PR DESCRIPTION
**Pull request by leem42**
PORTALS-660 Updates to search component
PORTALS-591 Changed endpoint logic to look for key in window with overrides, otherwise points to production
Got rid of css files!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,19 @@ Note there are a number of CDNs required to finish this functionality-
   <script src="https://unpkg.com/markdown-it-br@1.0.0/dist/markdown-it-br.min.js"></script>
 ```
 
+## Setting Endpoints
+
+### Configuring endpoint destinations for repo and portal
+Specifying the following in the window object will override request endpoints:
+
+```js
+window.SRC.OVERIDE_ENDPOINT_CONFIG = {
```
Comment (jay-hodgson) [inline README.md]: (sp) Override

File: src/lib/utils/getEndpoint.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export type BackendDestination = 'PORTAL_ENDPOINT' | 'SWC_ENDPOINT'
+// These variables indicate which endpoint should be called
+export const PORTAL_ENDPOINT: BackendDestination = 'PORTAL_ENDPOINT'
```
Comment (jay-hodgson) [inline src/lib/utils/getEndpoint.ts]: why do you have PORTAL_ENDPOINT and SWC_ENDPOINT, isn't that the same thing?  Expected PORTAL (or SWC) and REPO

File: src/lib/containers/Login.tsx [hunk 3]
```diff
@@ -47,104 +45,111 @@ class Login extends React.Component<Props, State> {
...
-              })
-            })
-            .catch((err: any) => {
-              console.log('Handle login failed with err = ', err)
-              this.setState({
-                errorMessage: err.reason,
-                hasLoginInFailed: true,
-                isSignedIn: false
-              })
+    SynapseClient.login(this.state.username, this.state.password)
```
Comment (conundrumgirl) [inline src/lib/containers/Login.tsx]: Just a general thought, I wonder if it makes sense to start refactoring  nested promises into  async/awaits when we are in those areas doing other changes.

File: src/lib/utils/getEndpoint.ts [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+  PORTAL: string
+}
+
+export const PRODUCTION_ENDPOINT_CONFIG: EndpointObject = {
+  REPO: 'https://repo-prod.prod.sagebase.org/',
+  PORTAL: 'https://www.synapse.org/',
+}
+
+// Given an endpoint will return the specific stack object
+export const getEndpoint = (endpoint: BackendDestination) => {
```
Comment (conundrumgirl) [inline src/lib/utils/getEndpoint.ts]: please add return type

File: src/lib/utils/getEndpoint.ts [hunk 5]
```diff
@@ -0,0 +1,33 @@
+export type BackendDestination = 'PORTAL_ENDPOINT' | 'SWC_ENDPOINT'
+// These variables indicate which endpoint should be called
+export const REPO_ENDPOINT: BackendDestination = 'PORTAL_ENDPOINT'
```
Comment (conundrumgirl) [inline src/lib/utils/getEndpoint.ts]: naming is a little confusing REPO_ENDPOINT = 'PORTAL_ENDPOINT' and PORTAL_ENDPOINT: BackendDestination = 'SWC_ENDPOINT' ?

File: src/lib/utils/getEndpoint.ts [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+export const PORTAL_ENDPOINT: BackendDestination = 'SWC_ENDPOINT'
+
+// Encodes the portal and repo service destinations
+type EndpointObject = {
+  REPO: string
+  PORTAL: string
+}
+
+export const PRODUCTION_ENDPOINT_CONFIG: EndpointObject = {
```
Comment (conundrumgirl) [inline src/lib/utils/getEndpoint.ts]: I wonder if these objects can just be distilled to an ENUM keyed off the BackendDestination  (may be the answer is 'no':)

File: src/lib/containers/Login.tsx (hunk 3 above)
Comment (leem42) [reply src/lib/containers/Login.tsx]: I like that idea too! I can change this over to use async/await

File: src/lib/utils/getEndpoint.ts (hunk 5 above)
Comment (leem42) [reply src/lib/utils/getEndpoint.ts]: whoops good catch, I will fix that..

**Pull request merged**